src/main/java/com/store/config/WebSecurityConfig.java

```diff
@@ -0,0 +1,53 @@
+package com.store.config;
+
+import lombok.AllArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.builders.WebSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.crypto.password.NoOpPasswordEncoder;
+
+import javax.sql.DataSource;
+
+
+@Configuration
+@EnableWebSecurity
+@AllArgsConstructor(onConstructor = @__(@Autowired))
+public class WebSecurityConfig extends WebSecurityConfigurerAdapter {
+    private static final String usersQuery = "select email, password, active from users where email=?";
```

Comment:
Naming is not right. 
https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names